ss58-registry.json

```diff
@@ -1234,6 +1234,15 @@
       "standardAccount": "*25519",
       "website": "https://bsx.fi"
     },
+    {
+      "prefix": 11159,
+      "network": "mosaic-chain-test",
+      "displayName": "Mosaic Chain Testnet",
+      "symbols": ["TMOS"],
+      "decimals": [18],
+      "standardAccount": "*25519",
+      "website": "https://mosaicchain.io"
+    },
```

Comment:
```suggestion
```

Test nets don't get their own prefix. They can just use 42 as they don't have any value

Reply:
Okay, I guess. We thought to maintain a long-term testnet on Paseo. All networks use 42 there?

Reply:
Yes. These networks have no economic value.